frontend/src/components/processes/ProcessDetailContent.js

```diff
@@ -55,16 +51,8 @@ const ProcessDetailContent = ({
       listProcesses({id__in: id});
     }
 
-    if(isLoaded && !childrenProcessesAreLoaded){
-      // Need to be queried as a string, not as an array in order to work with DRF filters
-      const processIDSAsStr = [id].concat(process.children_processes).join()
-      listProcesses({id__in:processIDSAsStr});
-    }
-
-    if (isLoaded && childrenProcessesAreLoaded && !allPropertiesAreLoaded) {
-      // Need to be queried as a string, not as an array in order to work with DRF filters
-      const processIDSAsStr = [id].concat(process.children_processes).join()
-      listPropertyValues({ object_id__in: processIDSAsStr, content_type__model: "process" });
+    if (isLoaded && !isProcessPropertiesLoaded(processesByID, propertyValuesByID, id)) {
```

Comment:
Any code which causes the redux state associated with the component to change needs to be wrapped in `useEffect`. Otherwise, we are modifying state during render, and that generates a react error.

Reply:
Huh. I was told it was unnecessary. That's good to know 👍🏼